### ISSUE TITLE
Survey links on workshop pages?

### ISSUE THREAD

**Issue by dvanic** [Sydney-Informatics-Hub/training-template-old#4]
Do we want links to the pre-workshop survey on the page? Aren't we capturing all in eventbrite?

**Comment by SIHAdmin** [comment]
These are SWC links. See comment to #7

**Comment by dvanic** [comment]
Yes, and even when I've run Carpentries, I've never actually used the Carpentry surveys (I've rebuilt my own using Google forms, to capture other relevant info ) - so these can be safely deleted. Don't know if we *need* to link to our survey in this page at all...

**Comment by SIHAdmin** [comment]
I should note that I've never seen these links actually rendered because I haven't made a workshop page! :)
